Directory.Build.targets

```diff
@@ -0,0 +1,11 @@
+<Project>
+
+  <ItemGroup Condition="'$(EnableStyleCopAnalyzers)' != 'false'">
+    <PackageReference Include="StyleCop.Analyzers" Version="1.1.0-beta006" PrivateAssets="all" />
+  </ItemGroup>
+
+  <ItemGroup>
```

Comment:
❗️ Need to disable this block for the Visual Studio projects or the wrong version will be used.

Reply:
➡️ Fixed and added a comment explaining why this special treatment is needed